src/xai_grok/schemas/tool_call_message.py

```diff
@@ -0,0 +1,9 @@
+import typing
+
+from xai_grok.schemas.message import Message
+from xai_grok.schemas.tool_call import ToolCall
+
+
+class ToolCallMessage(Message):
+    role: typing.Literal["assistant"]
```

Comment:
The `toolCalls` attribute uses camelCase naming which violates Python's snake_case convention for attributes. This will cause inconsistency with other Python code and should be renamed to `tool_calls`.